### ISSUE TITLE
Civilization 6 not saving game options

### ISSUE THREAD

**Issue by plOloV2** [canonical/steam-snap#322]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you checked the wiki for solutions?

- [X] I have searched the wiki

### Is this a Steam Snap-specific issue?

- [X] This is a Steam Snap-specific issue

### Current Behavior

On snap version of Steam I was unable to save game options in Civilization 6. `AppOptions.txt` file was not being created anywhere. On deb version everything works fine.

### Expected Behavior

I expected changes I made to be saved. Expected to find `AppOptions.txt` file in `~/.local/share/aspyr-media/Sid Meier's Civilization VI` folder.

### Environment

```yaml
Ubuntu 22.10
latest  Steam snap (I don't  have it installed now, but earlier today I made sure I have newest version before switching to deb)
I'm running Intel  NUC9 i9 Extreme with RTX3060ti.
```

**Comment by ashuntu** [comment]
I don't own Civilization 6, but have you tried checking the directories in the snap's directories? E.g. `~/snap/steam/common/.local/share` or `~/snap/steam/common/.steam/steam/steamapps/common`.

**Comment by plOloV2** [comment]
Yeah, I searched my whole disk for `AppOptions.txt` file but nothing was found.

**Comment by cmmrandau** [comment]
Same problem here. Can't even save game files locally, only cloud save works. After suspend cloud save doesn't work either. Had to keep the laptop running for a week in order to finish the game. 

Ubuntu 23.10, Steam snap, all updated.

**Comment by runninghai** [comment]
Same problem. It seem the game can not read&&write the local file system. Game options && saves can not be saved.

**Comment by alex-brook** [comment]
Is there an update on this?

I only realised there was an issue after the game crashed (!) and none of my saves were saved

**Comment by alexmc6** [comment]
Same problem here. Snap version of steam, ubuntu 23.04. brand new machine. 

I tried to create the directory, but that did not help 

mkdir -p  "~/.local/share/aspyr-media/Sid Meier's Civilization VI"

**Comment by berdal84** [comment]
Does anyone fixed this issue?

I tried the same as @alexmc6 (under ubuntu 22.04.4 though), but this does not help.

**Comment by djz88** [comment]
I have the same issue. Debian Trixie(snap steam 1.0.0.79 /191),  I have been able to track  it down to apparmor profile.

```
= AppArmor =
Time: Jun 29 17:48:10
Log: apparmor="DENIED" operation="mknod" class="file" profile="snap.steam.steam" name=$LONGALFANUM pid=297942 comm=43697636202857696E4944203229 requested_mask="c" denied_mask="c" fsuid=1000 ouid=1000
File: /home/user/.local/share/aspyr-media/Sid Meier's Civilization VI/AppOptions.txt (write)
Suggestions:
* adjust program to write to $SNAP_DATA, $SNAP_COMMON, $SNAP_USER_DATA or $SNAP_USER_COMMON
* add 'personal-files (see https://forum.snapcraft.io/t/the-personal-files-interface for acceptance criteria)' to 'plugs'
```
```

= AppArmor =
Time: Jun 29 17:48:10
Log: apparmor="DENIED" operation="mkdir" class="file" profile="snap.steam.steam" name=$LONGALPHANUM pid=297942 comm=43697636202857696E4944203229 requested_mask="c" denied_mask="c" fsuid=1000 ouid=1000
File: /home/user/.local/share/aspyr-media/Sid Meier's Civilization VI/Logs/ (write)
Suggestions:
* adjust program to write to $SNAP_DATA, $SNAP_COMMON, $SNAP_USER_DATA or $SNAP_USER_COMMON
* add 'personal-files (see https://forum.snapcraft.io/t/the-personal-files-interface for acceptance criteria)' to 'plugs'
```

**Comment by ashuntu** [comment]
This may be resolved with snapd 2.64. You can try by refreshing snapd with `snap refresh snapd --edge` (and restarting). **Make sure to go back to stable after testing** with `snap refresh snapd --stable`.
